src/views/Main.js

```diff
@@ -9,36 +9,68 @@
 import React, {useState, useEffect} from 'react';
 import useNFC from '../utils/nfc';
 import useUuid from '../utils/uuid';
-import {StyleSheet} from 'react-native';
+import {StyleSheet, View, Image, TouchableHighlight} from 'react-native';
 import {useStoreState, useStoreActions} from 'easy-peasy';
 import {Container, Content, Button, Text, H3, H1} from 'native-base';
+import { Col, Row, Grid } from 'react-native-easy-grid';
+import Scan from './Scan';
 
 const styles = StyleSheet.create({
+  headerImage: {
+    resizeMode: 'contain',
+    width: '100%',
+    height: '100%',
+    position: 'absolute',
+    zIndex: -1,
+    top: '-35%',
+  },
   content: {
-    margin: 20,
+    margin: '10%',
   },
   button: {
     marginVertical: 10,
     justifyContent: 'center',
+    alignSelf: 'center',
+    width: '70%',
+    height: '70%',
+    zIndex: -1,
+    position: 'absolute',
   },
   text: {
     textAlign: 'center',
+    color: 'white',
+    marginVertical: '7%',
   },
-  image: {
-    width: 30,
-    height: 30,
-    margin: 10,
-    marginLeft: 0,
-    marginRight: 0,
+  menuBtn: {
+    width: 145,
+    height: 145,
+    color: '#FFFFFF',
+    borderRadius: 2,
+    alignSelf: 'center',
+    shadowColor: "#000",
+    shadowOffset: {
+    	width: 0,
+    	height: 2,
+    },
+    shadowOpacity: 0.25,
+    shadowRadius: 3.84,
+    elevation: 10,
+    marginVertical: '10%',
+    bottom: '-30%',
   },
+  menuText: {
+    bottom: 70,
+    fontSize: 18,
+    textTransform: 'capitalize',
+  }
 });
 
 const Main = props => {
   //Set navigation options:
 
   const [text, setText] = useState('');
   const [currUuid, _generateUUID] = useUuid();
-  const {getNFC, _read, _write} = useNFC(currUuid, setText);
+  const {_read, _write} = useNFC(currUuid, setText);
```

Comment:
do you need `_read, _write` or even any of these hooks?
Also do you need to initialize the actions or store?

Reply:
That was my bad.